resources/es.ftl

```diff
@@ -0,0 +1,59 @@
+auth = Por favor usa el siguiente enlace para autenticarte en tu cuenta Wikimedia: [Autenticación]({$url})
+
+auth_exists_in_server = Actualmente estás autenticado en este servidor. No es necesario autenticarse de nuevo.
+
+auth_to_server = Estás identificado como [{$name}](<{$url}>). ¿Quieres autenticarte de nuevo en este servidor?
+yes = Sí
+no = No
+
+auth_footer = Este enlace tendrá validez por 5 minutos.
+
+authreq_canceled = Autenticación cancelada.
+authreq_expired = Autenticación expirada.
+
+authlog = {$mention} autenticado como [User:{$username}](<{$user_link}>) (id {$wmf_id}).
```

Comment:
Oh, also. If you don't want to use the `User:` prefix, you can remove it like `pl` did, or you could translate it into the prefix that your language uses.